docs/en/observability/synthetics-lightweight.asciidoc

```diff
@@ -55,28 +66,161 @@ heartbeat.scheduler:
-* {heartbeat-ref}/monitor-tcp-options.html[TCP monitors]
+* <<synthetics-lightweight-http>>
+* <<synthetics-lightweight-icmp>>
+* <<synthetics-lightweight-tcp>>
+
+The `tcp` and `http` monitor types both support SSL/TLS and some proxy settings.
+
+[discrete]
+[[synthetics-lightweight-common-options]]
+== Common options
+
+You can specify the following options when defining a synthetic monitor in any location.
+These options are the same for all monitors. Each monitor type has additional configuration
+options that are specific to that monitor type.
+
+// Reference table
+include::synthetics-reference/lightweight-config/common.asciidoc[]
+
+[discrete]
+[[synthetics-lightweight-http]]
+== HTTP options
+
+// Is this still relevant?
+The options described here configure Heartbeat to connect via HTTP and
+optionally verify that the host returns the expected response.
+
+Valid options for HTTP monitors include <<synthetics-lightweight-common-options,all common options>>
+and the following HTTP-specifc options:
+
+// Reference table
+include::synthetics-reference/lightweight-config/http.asciidoc[]
+
+[discrete]
+[[synthetics-lightweight-icmp]]
+== ICMP options
+
+// Is this still relevant?
+The options described here configure Heartbeat to use ICMP (v4 and v6) Echo
+Requests to check the configured hosts. Please note that on most platforms you
+must execute Heartbeat with elevated permissions to perform ICMP pings.
+
+// Is this still relevant?
+On Linux, regular users may perform pings if the right file capabilities are set. Run
+`sudo setcap cap_net_raw+eip /path/to/heartbeat` to  grant Heartbeat ping capabilities on Linux.
+Alternatively, one may grant ping permissions to the user Heartbeat runs as. To grant ping permissions
+in this way, run `sudo sysctl -w net.ipv4.ping_group_range='myuserid myuserid'`.
+
+// Is this still relevant?
+Other platforms may require Heartbeat to run as root or administrator to execute pings.
+
+Valid options for ICMP monitors include <<synthetics-lightweight-common-options,all common options>>
+and the following ICMP-specifc options:
+
+// Reference table
+include::synthetics-reference/lightweight-config/icmp.asciidoc[]
+
+[discrete]
+[[synthetics-lightweight-tcp]]
+== TCP options
+
+// Is this still relevant?
+The options described here configure Heartbeat to connect via TCP and
+optionally verify the endpoint by sending and/or receiving a custom payload.
+
+Valid options for TCP monitors include <<synthetics-lightweight-common-options,all common options>>
+and the following TCP-specifc options:
+
+// Reference table
+include::synthetics-reference/lightweight-config/tcp.asciidoc[]
+
+[discrete]
+[[synthetics-lightweight-data-types]]
+== Data types reference
+
+Values of configuration settings are interpreted as required by beats.
+If a value can not be correctly interpreted as the required type - for example a
+string is given when a number is required - the beat will fail to start up.
```

Comment:
Preview: [Data types reference](https://observability-docs_2729.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html#synthetics-lightweight-data-types)

Do you think copying over data types from Heartbeat docs (below) is helpful or should we just link to the Heartbeat docs instead?

Reply:
I think this is helpful, my general preference is not to link out to the heartbeat docs. This stuff hasn't changed in many years, so I'm not worried about duplication